queries/subsquid/general/nftByBlockNumber.graphql

```diff
@@ -0,0 +1,11 @@
+query nftByBlockNumber($limit: Int = 10, $blockNumber: BigInt) {
+  nftEntities(
+    limit: $limit
+    where: { blockNumber_eq: $blockNumber }
```

Comment:
can you have more than 1 nftEntity on a block?

Reply:
yes, when I test create nft with copies they had same block